### DISCUSSION TITLE
Possibility of loading tasks when the files are missing in the Json files

### DISCUSSION THREAD

**Discussion by aoloo-r** [StanfordVL/BEHAVIOR-1K#994] [Q&A]
Hello are these files available in the B1K dataset because whenever I run them  I constantly find an error and these are files are  ot available but this is a project that used to run like 8 months ago, so my question is that can I can't load the tasks then do you think  removed all those tasks so I can never load those tasks again during update , or probably there is change the way to load them -- meaning we have to make some modifications in my code:

Here are the tasks in line of code:

 a_name = "boil_water_in_the_microwave"
 config["scene"]["scene_model"] = "Beechwood_0_int"

a_name = "bringing_water"
config["scene"]["scene_model"] = "house_single_floor"

a_name = "cook_a_frozen_pie"
config["scene"]["scene_model"] = "Beechwood_0_int"

 a_name = "halve_an_egg"
config["scene"]["scene_model"] = "Benevolence_1_int"

a_name = "store_firewood"
config["scene"]["scene_model"] = "Ihlen_0_int"

**Comment by aoloo-r** [comment]
@cgokmen @ChengshuLi kindly help with this

**Reply by aoloo-r** [reply]
hello @hang-yin would you kindly help me with this

**Comment by hang-yin** [comment]
Hi @Roythescientist , if I'm understanding your question correctly, you're saying when you load some specific tasks in some specified scenes, you are seeing errors like `missing json files`. 

Currently, each of our BEHAVIOR 1K task has exactly one pre-sampled instance (each task correspond to exactly one scene). You can get a list of these pre-sampled task instances with `ls -l <gm.DATASET_PATH>/scenes/*/json/*task*`, detailed here: https://behavior.stanford.edu/omnigibson-develop/behavior_components/behavior_tasks.html. You can also use online_object_sampling to sample new task instances in new scenes, detailed here: https://behavior.stanford.edu/omnigibson-develop/behavior_components/behavior_tasks.html#sample-behavior-tasks. If you would like to know which scenes a specific task is compatible with, please check out our knowledgebase: https://behavior.stanford.edu/omnigibson-develop/behavior_components/behavior_knowledgebase.html. 

Let me know if this helps!

**Reply by aoloo-r** [reply]
When I change online_object_sample = True it says there is an attribute error

**Comment by aoloo-r** [comment]
@hang-yin kindly try running this https://github.com/keke-220/vlm-tamp on your end  and see if it works perfectly on your side

**Reply by aoloo-r** [reply]
run eval.py
